gprofiler/profilers/python.py

```diff
@@ -335,7 +335,9 @@ def start(self):
             wait_event(self._POLL_TIMEOUT, self._stop_event, lambda: os.path.exists(self.output_path))
         except TimeoutError:
             process.kill()
-            logger.error(f"PyPerf failed to start. stdout {process.stdout.read()!r} stderr {process.stderr.read()!r}")
+            stdout = process.stdout.read()  # type: ignore
```

Comment:
Why is it needed? Because `stdout` might be None, or because it might not be `TextIOWrapper`?

If the former - then I think that adding `assert process.stdout is not None` is preferred.